sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/IEDAdapter.java

```diff
@@ -359,38 +403,37 @@ public ControlBlock<? extends ControlBlock> createControlBlock(ControlBlock<? ex
         TServices tServices = currentElem.getServices();
         TServiceSettingsNoDynEnum cbNameAtt = controlBlock.getControlBlockServiceSetting(tServices);
```

Comment:
It seems that the wrong XPath is checked.
The issue requires to check `/SCL/IED/AccessPoint@name="XXXXXX"/Services/YYYSettings/@cbName` bu what is checked here is `/SCL/IED/Services/YYYSettings/@cbName`

Reply:
Fixed.